tokens.html

```diff
@@ -0,0 +1,313 @@
+	7
+	'''
+	copy = sorted(pool)
+	size = len(copy)
+	if size % 2 == 1:
+		return copy[(size - 1) / 2]
+	else:
+		return (copy[size/2 - 1] + copy[size/2]) / 2</code></pre></dd>
+
+		<dt><code>important</code></dt>
+		<dd>Anything that is important.
+		<pre><code class="language-css">body {
+	color: red !important;
+}</code></pre>
+		<pre><code class="language-markdown"># This is a heading. Headings are important.</code></pre></dd>
+
+		<dt><code>comment</code></dt>
+		<dd>Code comments.
+		<pre><code class="language-markup">&lt;!-- Here's a comment --&gt;
+&lt;style&gt;
+	/* Here's another comment */
+&lt;/style&gt;
+&lt;script&gt;
+// Here's yet another comment
+&lt;/script&gt;</code></pre></dd>
+
+		<dt><code>inserted</code></dt>
+		<dd>Added or modified line, mainly for diffs. In general, also the idea of something being increased.
+		<pre><code class="language-diff">+		// Calculate the area of the circle
+		let area = PI * (radius ** 2);</code></pre></dd>
+
+		<dt><code>deleted</code></dt>
+		<dd>Deleted line, mainly for diffs. In general, also the idea of something being decreased/removed.
+		<pre><code class="language-diff">-		// TODO: Remove this comment
+		let foo = 'bar';</code></pre></dd>
+
+		<dt><code>bold</code></dt>
+		<dd>Bolded text. Mostly found in document-markup languages.
+		<pre><code class="language-markdown">**I am bolded text!**</code></pre></dd>
+
+		<dt><code>italic</code></dt>
+		<dd>Italicised text. Mostly found in document-markup languages.
+		<pre><code class="language-markdown">*I am italicised text!*</code></pre></dd>
+
+		<dt><code>tag</code></dt>
+		<dd>The markup version of a keyword, to indicate when an element begins and ends.
+		<pre><code class="language-markup">&lt;p&gt;Hello World!&lt;/p&gt;</code></pre></dd>
+
+		<dt><code>attr-name</code></dt>
+		<dd>Kind of like a property of a markup tag.
+		<pre><code class="language-markup">&lt;video allowfullscreen controls&gt;
+	&lt;source src="hello_world.mp4" type="video/mp4" /&gt;
+&lt;/video&gt;</code></pre></dd>
+
+		<dt><code>attr-value</code></dt>
+		<dd>The value or argument that is passed to a property in a markup tag.
+		<pre><code class="language-markup">&lt;p id="greeting"&gt;Hello World!&lt;/p&gt;</code></pre></dd>
+
+		<dt><code>namespace</code></dt>
+		<dd>Used to provide uniquely named elements and attributes in XML documents.
+		<pre><code class="language-markup">&lt;html:p foo:bar="baz" foo:weee&gt;&lt;/html:p&gt;</code></pre></dd>
+
+		<dt><code>prolog</code></dt>
+		<dd>The first part of an XML document.
+		<pre><code class="language-markup">&lt;?xml version="1.0" encoding="utf-8"?&gt;
+&lt;svg&gt;&lt;/svg&gt;</code></pre></dd>
+
+		<dt><code>doctype</code></dt>
+		<dd>Document type declaration, specific to markup languages.
+		<pre><code class="language-markup">&lt;!DOCTYPE html&gt;
+&lt;html&gt;&lt;/html&gt;</code></pre></dd>
+
+		<dt><code>cdata</code></dt>
+		<dd>Character data, specific to markup languages. Deprecated as of DOM4.
```

Comment:
It may be deprecated for modern websites but it's still going strong in XML.

```suggestion
		<dd>Character data, specific to markup languages.
```